projects/secrets.md

```diff
@@ -0,0 +1,52 @@
+# CaraML Secret
+
+CaraML provides capability for storing user's secret in the platform. The secret can be used by the user within CaraML workfload such as batch prediction, router deployment, and pipeline.
```

Comment:
The secret can be used by the user within CaraML workfload

workfload seems a typo?